content/Cost/300_Labs/300_Optimization_Data_Collection/2_Deploy_Additional_Roles.md

```diff
@@ -42,7 +42,7 @@ Modules deployed in **OptimizationDataCollectionStack** (previous step) allow to
 
 5. Choose **Template is ready** and **Upload a template file** and upload the optimisation_read_only_role.yaml file you downloaded from above. Click **Next**.
 
-6. Call the Stack **OptimizationDataRoleStack**. In the Parameters section use the Cost Optimization Account ID that you deployed the OptimizationDataCollectionStack into for **CostAccountID**. Under available modules section select modules which you you selected in **OptimizationDataCollectionStack** deployment step. This CloudFormation StackSet will provision required roles for modules in linked accounts. Detailed description of each module can be found [here](../3_data_collection_modules)
+6. Call the Stack **OptimizationDataRoleStack**. In the Parameters section use the Optimization Account ID that you deployed the OptimizationDataCollectionStack into for **OptimizationAccountID**. Under available modules section select modules which you you selected in **OptimizationDataCollectionStack** deployment step. This CloudFormation StackSet will provision required roles for modules in linked accounts. Detailed description of each module can be found [here](../3_data_collection_modules)
```

Comment:
I dont really want to change these terms as these match the rest of the Labs when we talk about cost account. Know we have other data but these are in the cost section and all link. Want the phrasing to be constant